app/src/main/AndroidManifest.xml

```diff
@@ -40,13 +47,16 @@
         <activity
             android:name=".auth.LoginActivity"
             android:exported="true"
-            android:label="@string/title_activity_login"
```

Comment:
The displayed name of our application (in the app overview section on the phone) was "Log in" before I removed this line. I believe this is because adding the "label" line overrides the displayed name of the application (since the login activity is our start-up activity) and could not find a better fix than removing this line.

Reply:
Alright, this seems to be intended behavior: https://stackoverflow.com/questions/5397405/android-application-name-appears-as-activity-name.